.travis.yml

```diff
@@ -1,14 +1,21 @@
 ---
 language: java
+# skip the default install (mvn install -DskipTests=true) to speed up the build
+install: true
 script: mvn -q install -P examples
 jdk:
 - openjdk7
 matrix:
   include:
   - jdk: oraclejdk7
-    script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+    # skip the default install (mvn install -DskipTests=true) to speed up the build
```

Comment:
isn't that inherited from above? btw. on java8 you just wrote the comment without the actual skipping